architecture/core_objects/overview.adoc

```diff
@@ -5,7 +5,20 @@
 :icons:
 :experimental:
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+The following sections give some information about general OpenShift and
+Kubernetes architecture and terms. The
+link:../architecture/core_objects/kubernetes_model.html[Kubernetes Model]
+section outlines how the Kubernetes infrastructure is used in your OpenShift
+instance. The link:../architecture/core_objects/openshift-model.html[OpenShift
+Model] section outlines any OpenShift-specific information about your
+infrastructure.
```

Comment:
Changed to "The Kubernetes model topic provides information about important resources of your OpenShift instance, which is extended by the OpenShift model topic."

I think putting in "the following" might be specific to books or sections of the same topic.